build.gradle

```diff
@@ -39,6 +39,7 @@ dependencies {
     compile gradleApi()
     compile 'org.codehaus.groovy:groovy-all:2.4.7'
     compile group: "org.everit.json", name: "org.everit.json.schema", version: "1.1.1"
+    compile 'com.github.fge:json-schema-validator:2.2.6'
```

Comment:
I usually like more explicit way of specifying it (same as the above line):
`
compile group: 'com.github.fge', name: 'json-schema-validator', version: '2.2.6'`